programs/openbook-v2/src/state/market.rs

```diff
@@ -13,8 +14,17 @@ pub type MarketIndex = u32;
 #[account(zero_copy)]
 #[derive(Debug)]
 pub struct Market {
-    /// Admin who can close this market
-    pub admin: Pubkey,
+    /// Admin who can collect fees from the market
+    pub collect_fee_admin: Pubkey,
+    /// Admin who can manage the oracle
+    pub manage_oracle_admin: PodOption<Pubkey>,
```

Comment:
What is this for?

Reply:
This admin would be able to manage the oracle, which currently just means setting the stub oracle. I included it in case there was a use for it later (e.g., a change oracle instruction), but on second thought it seems that it should be removed from this PR and included later only if there's a defined use for it.

Reply:
Correct